### ISSUE TITLE
Only one page in Nearby list - "Load more pages" does not have any effect

### ISSUE THREAD

**Issue by Matze2** [cgeo/cgeo#6705]
##### Detailed steps causing the problem:
- Open Nearby list
- First 20 entries are shown
- Press "Load more pages"
- After some time still only 20 entries are shown

##### Actual behavior after performing these steps:
It is not possible to load more than one "Nearby" page.

##### Expected behavior after performing these steps:
"Load more pages" should load more pages. Probably also an effect of website changes.

##### Version of c:geo used:
2017.08.22-NB-3917990

##### Is the problem reproducible for you?
Yes

##### System information:
```
--- System information ---
Device: D5503 (D5503, Sony)
Android version: 5.1.1
Android build: 14.6.A.1.236
c:geo version: 2017.08.22-NB-3917990
Google Play services: disabled - 11.3.02 (236-161239932)
Low power mode: inactive
Compass capabilities: yes
Rotation vector sensor: present
Orientation sensor: present
Magnetometer & Accelerometer sensor: present
Direction sensor used: orientation
Hide own/found: true
Map strategy: detailed
HW acceleration: enabled (default state)
System language: de_DE
System date format: dd.MM.yyyy
Debug mode active: no
System internal c:geo dir: /data/data/cgeo.geocaching (2,2 GB free) internal
User storage c:geo dir: /storage/emulated/0/cgeo (2,2 GB free) external non-removable
Geocache data: /storage/sdcard1/Android/data/cgeo.geocaching/files/GeocacheData (7,7 GB free) external removable
Database: /storage/emulated/0/Android/data/cgeo.geocaching/files/databases/data (39,7 MB) on user storage
Fine location permission: granted
Write external storage permission: granted
Geocaching sites enabled:
   geocaching.com: Logged in (Anmeldung OK) / BASIC
Geocaching.com date format: MM/dd/yyyy
Installed c:geo plugins:  none
--- End of system information ---
```

**Comment by Lineflyer** [comment]
I retested this as basic member with 2017.07.24 release and had no problem to load more caches.
There is probably another reason unrelated to the website change.

For what coordinates did you try the nearby search?

**Comment by kumy** [comment]
I can reproduce on 2017.07.24

**Comment by Matze2** [comment]
This also didn't work on Sunday afternoon CEST.

Then it worked again. Probably in the afternoon and on Monday (but not really sure about the time period).

Since Tuesday morning it does not work anymore.

I don't want to post my home coordinates here. But I tried with "Caches around" GC78Y21 which should use the same URL. Also here only 20 entries.

**Comment by Lineflyer** [comment]
@Matze2 
In case you are c:geo beta user on Google Play a new version should be available in the next minutes or hours. Maybe you can try this one and see if the search problem is still present.

**Comment by Matze2** [comment]
Sorry, I am a nightly build user. But as soon as the corresponding fix is available as NB I can try.

**Comment by kumy** [comment]
As I have merged release into master we can easily trigger a NB now. @Liflyer may you do it please. I m off my keyboard ;-) 

On August 22, 2017 10:34:46 PM GMT+02:00, Matze2 <notifications@github.com> wrote:
>Sorry, I am a nightly build user. But as soon as the corresponding fix
>is available as NB I can try.
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/cgeo/cgeo/issues/6705#issuecomment-324143932

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by Lineflyer** [comment]
Yup.
On its way.

**Comment by Lineflyer** [comment]
New nightly is available for download.

**Comment by Lineflyer** [comment]
For documentation:
http://ci.cgeo.org/view/website/job/website_GC_page-load-more/ can be used by the team to test availability of search page. Thanks @kumy

**Comment by Matze2** [comment]
So I have now 2017.08.22-NB1-2f1a321 and... THUMBS UP!
Well done guys!

**Comment by kumy** [comment]
Yeah!

**Comment by Lineflyer** [comment]
@Matze2 
Thanks for feedback!

**Comment by Matze2** [comment]
I have to say thank you for the light-speed-fix.

BTW: Because I just tried it, #5498 is still valid.
Just if somebody is bored ;)
But it's really minor.

**Comment by Lineflyer** [comment]
Should be solved by 1e506cc

**Comment by kumy** [comment]
I confirm. Fixed.

On August 23, 2017 10:04:09 AM GMT+02:00, Lars <notifications@github.com> wrote:
>Should be solved by 1e506cc
>
>
>
>
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/cgeo/cgeo/issues/6705#issuecomment-324253404

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
